package.json

```diff
@@ -26,11 +26,15 @@
   },
   "homepage": "https://github.com/tc39/proposal-record-tuple#readme",
   "dependencies": {
+    "@tc39/ecma262-biblio": "^2.1.2336",
     "browser-sync": "^2.26.7",
     "concurrently": "^5.2.0",
-    "ecmarkup": "^9.1.0",
-    "mkdirp": "^0.5.1",
+    "ecmarkup": "^13.0.1",
     "fs-extra": "^9.0.1",
-    "marked": "^4.0.10"
+    "marked": "^4.0.10",
+    "mkdirp": "^0.5.1"
+  },
+  "overrides": {
+    "nwsapi": "2.2.0"
```

Comment:
workaround while there is a transitive dependency bug

https://github.com/tc39/ecmarkup/pull/471#issuecomment-1183617260